### DISCUSSION TITLE
Organisation temps de travail

### DISCUSSION THREAD

**Discussion by Marine1719** [BIO6032/2022-SEIR-reproduction#21] [General]
Salut tout le monde! Je voudrais suggèrer une façon qu'on adapte nos temps de travail car je vois que personne n'a eu beaucoup de temps pour avancer sur le projet (moi y compris). 

Personnellement, je pars pour le canada dans 3 semaines et j'ai énormément de choses à faire incluant un gros déménagement pour l'étranger donc la dernière semaine avant mon départ j'aurais vraiment peu de temps. et j'aimerais donc avancer un maximum dessus avant (si possible bien sur). 

Du coup, je voulais proposer de par exemple partager un lien zoom (ou par discord) une fois/semaine hors des heures de cours pour essayer de travailler un maximum sur le projet ensemble pour ceux que ça intéresse. Même ne serait-ce qu'une heure pour avancer efficacement après les heures de cours. Si ça vous dit est-ce que vous êtes d'accord de commenter "oui" (ou non, sans obligation, je sais que certaines personnes préfèrent travailler individuellement) et ensuite on fait une discussion entre nous pour fixer un créneau horaire ? Merci :)

**Comment by salomebsk** [comment]
Salut Marine! Oui je pense que c'est une bonne idée. 
De mon côté, je gère actuellement des problèmes personnels qui ralentissent énormément tous mes projets et études. C'est plutôt au début du projet que j'ai pu être moins présente. Ce rendez-vous hors des heures de cours serait le bienvenu pour s'assurer d'avancer le plus efficacement et rapidement possible. 
Je suis d'accord! :)

**Comment by blanchette31** [comment]
Je crois que c'est une bonne idée également, ça n'as pas besoin de nécessairement être d'organiser un moment pour travailler tous ensemble. On peut faire ça, mais ça va peut-être être compliqué à planifier avec les horaires de tout le monde. Ça pourrait être juste de dire qui va travailler sur quoi entre les cours pour avancer le travail.

**Comment by Marine1719** [comment]
Ok super :)! Oui comme vous préferez soit on planifie des sessions pour avancer ensemble en live soit on se fixe des objectifs sinon je pense qu'on avancera pas efficacement.. @blanchette31 c'est juste que de nouveau planifier qui va faire quoi est compliqué si tout le monde ne participe pas... Et je pense que ça peut nous mettre justement un coup de pression de se voir une heure en live pour travailler ensemble, comme ça on s'oblige à être à jour. 

On peut essayer de fixer une première session au moins; est-ce qu'il y a des jours où vous n'êtes pas du tout disponibles? Moi le plus facile ce sont les lundis et vendredi :)

**Reply by aushay** [reply]
Je suis partante aussi. Si on décide effectivement de travailler ensemble (tous ou par sous-division de groupe), on pourrait éventuellement remplir ce document https://www.when2meet.com/?15144402-Kn4F9
